### DISCUSSION TITLE
Requesting to join team

### DISCUSSION THREAD

**Discussion by CodeChas3r1** [ROM-builders/temporary#21594] [General]
My successful builds
AlphaDroid
https://cirrus-ci.com/build/5862169296764928

crDroidAndroid
https://cirrus-ci.com/build/5020379161296896

BootleggersROM
https://cirrus-ci.com/build/4878323621298176

DerpFest
https://cirrus-ci.com/build/6296787774865408

Device Tree
https://github.com/CodeChas3r1/device_xiaomi_rosemary

Kernel Tree
https://github.com/CodeChas3r1/kernel_xiaomi_mt6785

Vendor Tree
https://github.com/CodeChas3r1/vendor_xiaomi_rosemary

My github username
https://github.com/CodeChas3r1

local_manifest
https://github.com/CodeChas3r1/local_manifest

Telegram Name & Username: Chas ( @CodeChas3r )

**Comment by Apon77** [comment]
Please check #learn instruction from telegram group.

**Reply by CodeChas3r1** [reply]
Ok sir thanks
